test/test_parallel_executor.cpp

```diff
@@ -110,72 +110,18 @@ TEST(ParallelExecutor, CompilerTest) {
 
   std::vector<Tensor> tf_input_tensors{x, y};
   shared_ptr<ngraph::runtime::Executable> ng_exec;
-
+  shared_ptr<PipelinedTensorsStore> pts;
+  std::string ser_ng_function;
   // Call the Executor to compile the funcion
   bool cache_hit = false;
-  ASSERT_OK(executor.GetNgExecutable(tf_input_tensors, ng_exec, cache_hit));
+  ASSERT_OK(executor.GetExecutableFunctionAndTensors(
+      tf_input_tensors, ng_exec, ser_ng_function, pts, cache_hit));
   ASSERT_FALSE(cache_hit);
 
   // Now call again to test that the cache works
-  ASSERT_OK(executor.GetNgExecutable(tf_input_tensors, ng_exec, cache_hit));
-
-  // If the cache doesn't work then the following will fire
+  ASSERT_OK(executor.GetExecutableFunctionAndTensors(
+      tf_input_tensors, ng_exec, ser_ng_function, pts, cache_hit));
   ASSERT_TRUE(cache_hit);
-
-  // Now validate that the nGraph function is available
-  std::shared_ptr<ngraph::Function> ng_function;
-  ASSERT_EQ(executor.GetNgFunction(ng_exec, ng_function),
-            tensorflow::Status::OK());
-
-  // Validate the nGraph Function
-  const auto& parameters = ng_function->get_parameters();
-  ASSERT_EQ(2, parameters.size());
-}
-
-TEST(ParallelExecutor, PipelinedTensorCreate) {
```

Comment:
This test is not needed anymore, as I got rid of GetTensorPipeline()

Reply:
Looks good, we can remove this I think

But refer to my comment above about an end-to-end test with 3 threads that is expected to fail